README.md

```diff
@@ -1 +1,14 @@
-# local_match
\ No newline at end of file
+# local_match
+
+
+## Pre-commit instructions
+#### In yout repository use the terminal and type
+
+#### **$ cd .git/hooks**
```

Comment:
I think it would be better if you create a simple script with these commands instead of raw instructions. What do you think?

Reply:
I like this idea 👍